ServerCore/Properties/PublishProfiles/PuzzleServerTestDeploy - Web Deploy.pubxml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You seem to have two of these. Do we actually need both?

Reply:
Nope, good catch, thanks! :)